packages/vats/src/lib-board.js

```diff
@@ -267,8 +270,7 @@ export const prepareBoardKit = baggage => {
          * `value` for its entire lifetime. For a well-known board, this is
          * essentially forever.
          *
-         * @param {PassableCap} value
-         * @throws if `value` is not a Key in the @agoric/store sense
```

Comment:
removed this because it shouldn't come as a surprise to get an error when the argument doesn't satisfy the param type